k8s/distributed-chroma/templates/sysdb-service.yaml

```diff
@@ -84,3 +92,16 @@ subjects:
   namespace: {{ .Values.namespace }}
 
 ---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: sysdb-serviceaccount-lease-watcher-binding
+  namespace: {{ .Values.namespace }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: lease-watcher
```

Comment:
discussed offline to move the definition of lease-watcher role out of logservice.yaml